### DISCUSSION TITLE
ModuleNotFoundError: Module not found: Error: Can't resolve 'fs'

### DISCUSSION THREAD

**Discussion by abelardolg** [vercel/next.js#15780] [Help]
# Bug report
I was following the tutorial when the app showed this  message.
## Describe the bug
I updated the [id].js file with the new version of the Post component mentioned [ at this page]. (https://nextjs.org/learn/basics/dynamic-routes/polishing-post-page)
Once I saved the file this message appeared in the browser.

## To Reproduce
Update the [id].js file and then save it.

Steps to reproduce the behavior, please provide code snippets or a repository:

## System information

- OS: [e.g. macOS, Windows] Ubuntu 19
- Browser (if applies) [e.g. chrome, safari] Chrome
- Version of Next.js: [e.g. 6.0.2]  The latest
- Version of Node.js: [e.g. 10.10.0] 14.5.0

**Comment by jamesmosier** [comment]
Hi @abelardolg. Do you have a repository you can share with this issue?

**Comment by abelardolg** [comment]
Hi @jamesmosier , here: https://github.com/abelardolg/nextjsblog

**Comment by MoOx** [comment]
If you have to use workaround is to use `config.resolve` to "disable" some modules that trigger the issue, you are probably not following the ["rule" of writing Node code in `pages` folder only](https://github.com/vercel/next.js/issues/16153#issuecomment-673374488).
